Project.ConsoleApp.Test/ConsoleAppTest.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+
+namespace Project.ConsoleApp.Test
+{
+    [TestClass]
+    public class ConsoleAppTest
+    {
+        [TestMethod]
+        public void Arguments_WithValidArguments()
+        {
+            //arrange
+            string file = CreateTempFile(Array.Empty<string>());
+
+            try
+            {
+                //act
+                Program.Main(new string[] { "-json", file, "-xml", file });
+                
+            }
+            catch (ArgumentException aex)
+            {
+                //assert
+                Assert.Fail(aex.Message);
+            }
+            finally
+            {
+                DeleteTempFile(file);
+            }
+        }
+
+        [TestMethod]
+        public void Arguments_WithInvalidArguments() => Assert.ThrowsException<ArgumentException>(() => Program.Main(new string[] { "json", string.Empty, "xml", string.Empty }));
+
+        [TestMethod]
+        public void Arguments_WithTooManyArguments()
+        {
+            //arrange
+            string file = CreateTempFile(Array.Empty<string>());
+
+            try
+            {
+                //act
+                Program.Main(new string[] { "-json", file, "-xml", file, "-test", file });
+            }
+            catch (ArgumentException aex)
```

Comment:
`Observation`: The act of throwing an exception will break the test. You don't need to be explicit about it.